views/quote.py

```diff
@@ -14,6 +14,6 @@ def get_quote(cls, theme, author):
         if error_message:
             return json.dumps({"error_message": error_message}), status
         if len(response) == 0:
-            return json.dumps({"error_message": "No quotes found"}), 501
+            return json.dumps({"response": None}), status
         index = randint(0, (len(response) - 1))
```

Comment:
I thought this would be in controller, but I guessed wrong. I didn't read carefully enough. no logic, (picking a random one from the response) should be in the views. views are for filtering the input and maybe differentiating between errors and expected behavior. Your abstraction is weak and leaking. Read up on what MVC is and practical examples of it.